internal/cmd/iam/command_groupmapping_update.go

```diff
@@ -0,0 +1,74 @@
+package iam
+
+import (
+	"github.com/spf13/cobra"
+
+	ssov2 "github.com/confluentinc/ccloud-sdk-go-v2/sso/v2"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+)
+
+const groupMappingNoOpUpdateErrorMsg = "one of `--description`, `--name`, or `--filter` must be set"
+
+func (c *groupMappingCommand) newUpdateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "update <id>",
+		Short:             "Update a group mapping.",
+		Args:              cobra.ExactArgs(1),
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validArgs),
+		RunE:              c.update,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Update the description of group mapping "pool-123456".`,
+				Code: `confluent iam group-mapping update pool-123456 --description "Updated description"`,
```

Comment:
Nit: Keep everything lowercase, easier to type!
```suggestion
				Code: `confluent iam group-mapping update pool-123456 --description "updated description"`,
```